pop-api/src/v0/assets/mod.rs

```diff
@@ -222,7 +220,7 @@ pub fn token_symbol(id: AssetId) -> Result<Vec<u8>> {
 
 #[inline]
 pub fn token_decimals(id: AssetId) -> Result<u8> {
-	ChainExtensionMethod::build(u32::from_le_bytes([V0, READ_STATE, ASSETS, TOKEN_DECIMALS]))
+	build_read_state(ASSETS, TOKEN_DECIMALS)
```

Comment:
We could have another set of build_dispatch and build_read_state functions defined at the v0::assets module scope to remove the need for the repeated ASSETS constant use.

Essentially providing the same functions at each level, narrowing the scope from general to somewhat specific.